python/spacewalk/common/rhnLog.py

```diff
@@ -309,15 +309,15 @@ def _exit():
 atexit.register(_exit)
 
 
-def log_level_to_logging_constant(rhnLog_log_level: int):
+def log_level_to_logging_constant(rhnlog_log_level: int):
```

Comment:
`rhnLog` is still called `rhnLog`, the name was chosen on purpose to reflect that.

Reply:
Ok, I'll revert this. It was raised as an issue by SonarCloud, but I agree it makes sense in this case to keep the `rhnLog_log_level` name.